addons/stock/models/stock_location.py

```diff
@@ -76,6 +76,13 @@ def _onchange_usage(self):
         if self.usage not in ('internal', 'inventory'):
             self.scrap_location = False
 
+    @api.constrains('location_id', 'usage')
+    def _check_location_transit(self):
```

Comment:
I understand the idea, but we do not add this kind of constrain in stable version. We never know if someone use this behavior

Reply:
@Whenrow I have remove the constrains, and add tests